.vscode/cspell.json

```diff
@@ -73,7 +73,10 @@
     "sdk/keyvault/azure-keyvault-administration/**",
     "sdk/formrecognizer/azure-ai-formrecognizer/**",
     "sdk/keyvault/azure-keyvault-secrets/**",
-    "sdk/identity/azure-identity/**",
+    "sdk/identity/azure-identity/azure/identity/_internal/linux_vscode_adapter.py",
+    "sdk/identity/azure-identity/azure/identity/_internal/win_vscode_adapter.py",
+    "sdk/identity/azure-identity/tests/pod-identity/**",
+    "sdk/identity/azure-identity/tests/managed-identity-live/service-fabric/**",
```

Comment:
Why are these files being excluded?

Reply:
Moved linux_vscode_adapter.py and win_vscode_adapter.py out of ignorePaths.